Src/PMS.DAL/ProductDAO.cs

```diff
@@ -84,6 +84,38 @@ public static List<ProductDTO> GetAllProducts(Boolean pLoadComments=false)
             }
         }
 
+        public static List<ProductDTO> GetPriceRangedProducts(int from, int to, Boolean pLoadComments = false)
+        {
+            var query = "Select * from dbo.Products Where IsActive = 1 AND (Price Between '"+from+"' AND '"+to+"');";
```

Comment:
We try to use stored procedure as much as possible. Accepting this for now.

Reply:
Whenever we select a filter, it keeps on appending the items, which causes repetition. for example price of a product is 100, now if i chose all products it shows me the product.Now when i chose a range lets say 0-5000 it appends the previous item that causes duplication @bilalshahzad139 @abdullahaleem355

Reply:
I've just pulled develop and checked. It is working fine. I am removing the previous childs by $("tblBody").empty.
You can check. If still there is this issue. Let me know bilal.

Reply:
> I've just pulled develop and checked. It is working fine. I am removing the previous childs by $("tblBody").empty.
> You can check. If still there is this issue. Let me know bilal.
Please change this $("tblBody").empty to $("productsDiv").empty. It will work fine.